Morphs/Importers/RootPaths.props

```diff
@@ -14,10 +14,6 @@
     <Import Condition="'$(CanvasRootDirectory)' == ''" Project="CanvasRootDirectory not defined. Define this property before importing $(MSBuildThisFile)." />
   </ImportGroup>
```

Comment:
This PropertyGroup deletion is breaking the Deploy.ps1 script.  From what I've found, updating this with an updated v142 value or modifying the Deploy script to not include the compiler in the lib paths should fix this.

Reply:
I'm not sure I understand why these scripts are here. Who uses them?

Reply:
I believe they were a carryover from when we were building nuget packages.  TBH, we can just cut the script call from the Transcoders post-build.